wangle/acceptor/TransportInfo.h

```diff
@@ -104,18 +121,20 @@ struct TransportInfo {
    */
   int64_t ssthresh{-1};
 
-#if defined(__linux__) || defined(__FreeBSD__)
+#if defined(__linux__) || defined(__FreeBSD__) || defined(__APPLE__)
   /*
    * TCP information as fetched from getsockopt(2)
    */
   tcp_info tcpinfo {
```

Comment:
Looks like this can just be:

```
tcp_info tcpinfo{};
```

Reply:
Yes, and that way all three cases are handled. I don't know why this was done like that, but I followed the existing pattern.